static/js/directives/ui/logs-view.js

```diff
@@ -892,3 +916,8 @@ angular.module('quay').directive('logsView', function () {
 
   return directiveDefinitionObject;
 });
+
+function obfuscate_email(email) {
```

Comment:
Should we obfuscate the email? If we want the ability to audit log exports shouldn't the user be able to see which email it was sent to?

Reply:
I'm generally not in favour of printing out e-mails in raw text format in browsers, due to potential malware being present on computers which can read screens. The e-mail address is stored as part of the log in the `logentry3` table and would be exported in logs as well. It just wouldn't be shown as clear text in the UI. The full information is still available via API:

```
    {
      "kind": "export_logs_success",
      "metadata": {
        "export_id": "0f38180f-0e11-48c5-97f1-48cfdbfa179e",
        "url": null,
        "email": "some@email.com"
      },
      "ip": "172.17.0.1",
      "datetime": "Mon, 19 Aug 2024 22:11:35 -0000",
      "performer": {
        "kind": "user",
        "name": "ibazulic",
        "is_robot": false,
        "avatar": {
          "name": "ibazulic",
          "hash": "9b6277f06ccaf5ee50f00e85ade1ab0a7210c13c7e7c5739d1f318d4feca183c",
          "color": "#a1d99b",
          "kind": "user"
        }
      }
    },
```